web/web_test.go

```diff
@@ -33,4 +35,60 @@ func TestDownloadFile(t *testing.T) {
 	require.NoError(t, err)
 
 	assert.Equal(t, expectedContent, data)
+
+	tests := []struct {
+		name        string
+		url         string
+		out         string
+		expectError bool
```

Comment:
nit: these fields are public to the struct so they should start with capital letters. Doesn't really matter in the context of tests, but it's a good habit to maintain that should make refactors easier in the future.